model/mention.go

```diff
@@ -15,12 +15,15 @@ type Mention struct {
 	kallax.Timestamps `kallax:",inline"`
 	// Endpoint is the repository URL as found.
 	Endpoint string
+	// Aliases are all the endpoints obtained from this mention. Endpoint field should be also included
+	Aliases []string
+	// IsFork is setted if the provider know if this Mention provides from a forked repository or not.
```

Comment:
Suggested rewording `IsFork is set to true if the repository is known to be fork. It is set to nil if the provider does not provide this information at all. Note that false means that the repository is not a known fork to the provider, but it might still be a fork, for example, a fork in GitHub from an original repository in Bitbucket.`